src/test/java/reposense/parser/AuthorConfigParserTest.java

```diff
@@ -183,4 +186,30 @@ public void parse_multipleColumnsWithCommasAndDoubleQuotes_success() throws Exce
             Assert.assertEquals(AUTHOR_ALIAS_COMMAS_AND_DOUBLE_QUOTES_MAP.get(author), author.getAuthorAliases());
         });
     }
+
+    @Test
+    public void parse_multipleRepos_success() throws Exception {
+        AuthorConfigCsvParser authorConfigCsvParser =
+                new AuthorConfigCsvParser(AUTHOR_CONFIG_MULTIPLE_REPOS);
+        List<AuthorConfiguration> configs = authorConfigCsvParser.parse();
+
+        Assert.assertEquals(4, configs.size());
+        AuthorConfiguration config = configs.get(0);
```

Comment:
perhaps can make use of the constants created in `RepoLocation`?

Reply:
Those constants would not apply here, yes?
In this test case, we are parsing a `author-config.csv` file, which gives us a list of `AuthorConfiguration` objects.

The rest of the test case simply asserts various properties for each `AuthorConfiguration` object in the list.

Reply:
Yep my bad.